app/root.tsx

```diff
@@ -88,13 +88,21 @@ const inlineThemeCode = stripIndents`
   }
 `;
 
+const analyticsCode = stripIndents`
```

Comment:
Strong suggestion: Let's move this to a separate file with comments explaining what this is and links to the analytics dashboard and also the ability to re-generate this when necessary